modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java

```diff
@@ -468,7 +468,7 @@ protected void validateFile(
 			throw new LARFileException(LARFileException.TYPE_MISSING_MANIFEST);
 		}
 
-		Element rootElement = null;
+		Element rootElement;
```

Comment:
@JanisZhang Same question as the other PR..

Reply:
@dantewang Noted, thanks.